src/jenkins.py

```diff
@@ -308,18 +310,37 @@ def _unlock_wizard(container: ops.Container) -> None:
     )
 
 
-def _install_configs(container: ops.Container) -> None:
+def _install_jenkins_config(container: ops.Container, filename: str) -> None:
     """Install jenkins-config.xml.
 
     Args:
         container: The Jenkins workload container.
+        filename: the source file to copy contents from.
     """
-    with open("templates/jenkins-config.xml", encoding="utf-8") as jenkins_config_file:
+    with open(filename, encoding="utf-8") as jenkins_config_file:
         container.push(CONFIG_FILE_PATH, jenkins_config_file, user=USER, group=GROUP)
+
+
+def _install_configs(container: ops.Container) -> None:
+    """Install jenkins-config.xml and logging files.
+
+    Args:
+        container: The Jenkins workload container.
+    """
+    _install_jenkins_config(container, "templates/jenkins-config.xml")
     with open("templates/logging.properties", encoding="utf-8") as jenkins_logging_config_file:
         container.push(LOGGING_CONFIG_PATH, jenkins_logging_config_file, user=USER, group=GROUP)
 
 
+def install_auth_proxy_config(container: ops.Container) -> None:
+    """Install jenkins-config.xml for auth_proxy.
+
+    Args:
+        container: The Jenkins workload container.
+    """
+    _install_jenkins_config(container, "templates/reverse-proxy-auth-jenkins-config.xml")
```

Comment:
Minor/nitpicking: could "templates/<file name>" be constants?

Reply:
Updated